Doc/reference/compound_stmts.rst

```diff
@@ -378,8 +378,8 @@ exception group with an empty message string. ::
    ...
    ExceptionGroup('', (BlockingIOError()))
 
-An :keyword:`!except*` clause must have a matching type,
-and this type cannot be a subclass of :exc:`BaseExceptionGroup`.
+An :keyword:`!except*` clause must have a matching type or a tuple of matching
```

Comment:
It might be worth expanding this further:

- The docs for `except` (the previous section) never say that each type must be a subtype of `BaseException`. That's probably worth adding.
- One important difference between `except` and `except*` is that the former allows "bare" except clauses (just `except:`), but `except*` doesn't. I think that's worth mentioning in a sentence by itself.
- We don't say what happens if the type is a subclass of BaseExceptionGroup.

So I'd suggest something like this:

```suggestion
Unlike :keyword:`except` clauses, :keyword:`!except*` clauses must have a matching
expression; `except*:` is not valid syntax. When the handler is evaluated, the
expression must evaluate to a type or a tuple containing types. Each type must be a
subclass of :exc:`BaseException` and must not be a subclass of :exc:`BaseExceptionGroup`.
```

Reply:
Okay, I've added a mention that the runtime will raise a TypeError if the type is wrong, and updated the `except` section to have similar language.

Should the latter be split into a separate PR, or should I just update the title/summary of this PR?